src/main/resources/assets/opencomputers/recipes/default.recipes

```diff
@@ -244,6 +244,11 @@ batteryUpgrade3 {
           ["oc:capacitor", chipDiamond, "oc:capacitor"]
           [nuggetIron, "oc:capacitor", nuggetIron]]
 }
+beekeeperUpgrade {
```

Comment:
So far ik we use gt recipe sheet. Not sure if this recipe will be added like this or we should add it via script? @repo-alt

Reply:
I will add a recipe via assembler